db_scripts/mosip_regprc/ddl/regprc-registration.sql

```
@@ -68,3 +68,5 @@ CREATE INDEX IF NOT EXISTS idx_workflow_active_true ON regprc.registration USING
 
 CREATE INDEX IF NOT EXISTS idx_registration_workflow_instance_id ON regprc.registration(workflow_instance_id desc);
 CREATE INDEX IF NOT EXISTS idx_registration_sts_resume ON regprc.registration(status_code, resume_timestamp, default_resume_action);
+
+CREATE INDEX CONCURRENTLY idx_registration_reprocessor ON regprc.registration (latest_trn_status_code, latest_trn_dtimes, reg_process_retry_count, status_code);
```

SQLFluff (4.1.0)

[error] Line 36, Position 1: Found unparsable section: 'CREATE INDEX IF NOT EXISTS idx_reg_lates...'

(PRS)